tide_core.install

```diff
@@ -212,3 +212,18 @@ function tide_core_update_8005() {
   $config_location = [drupal_get_path('module', 'tide_core') . '/config/optional'];
   _tide_import_single_config('prlp.settings', $config_location);
 }
+
+/**
+ * Enable the blockquote template available to all Node types.
+ */
+function tide_core_update_8006() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_publication') . '/config/install'];
```

Comment:
@vincent-gao @sonnykt As my understanding, this issue impacted on all content type except `landing_page` and `page`. So should we fix it for `news`, `grants` and all content types in this PR as well?

Reply:
@tim-yao the above code is not correct.

Reply:
@sonnykt @tim-yao I will move this code snippet to the publication module. 
this module will keep the `config/install/wysiwyg_template.wysiwyg_template.blockquote.yml`
and the update hook will be moved to `publication` module